root/app/nginx-ldap-auth-daemon.py

```diff
@@ -94,9 +94,15 @@ def do_GET(self):
             user, passwd = auth_decoded.split(':', 1)
         except InvalidToken:
             self.log_message('Incorrect token. Trying to decode credentials from BASE64...')
-            auth_decoded = base64.b64decode(auth_header[6:])
-            auth_decoded = auth_decoded.decode("utf-8")
-            user, passwd = auth_decoded.split(':', 1)
+            # Wrapping below in try-except block to catch failed decoding due to expired cookie
+            try:
```

Comment:
Try/catch inside another try/catch is a bit weird, you can extract the logic into 2 helper functions, each with its own try/catch.